nixos/modules/services/hardware/monado.nix

```diff
@@ -0,0 +1,73 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+let
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkDefault mkEnableOption mkPackageOption;
```

Comment:
this no longer works due to the removal of mkDefault: https://github.com/NixOS/nixpkgs/commit/fb4fc93a3eacab755184924ae7bbb3d1e28e5575

Reply:
What would the installation cd image have to do with this?

Reply:
indeed, it seems like i've just pulled out a random commit with a seemingly-relevant message. my bad!

still, this did not work for me due to `lib.options.mkDefault` not existing. it should be updated to the current whereabouts of this function.

Reply:
Where did you even pull this rando commit from? :D

`mkDefault` and the like are in `lib`. Though honestly, they should probably be in `lib.options` aswell.

Reply:
Rebased on latest master and fixed this. Not sure why it was that way. Might have been me making last minute changes without testing.